new-lamassu-admin/src/pages/Cashout/WizardSplash.js

```diff
@@ -2,29 +2,47 @@ import { makeStyles } from '@material-ui/core'
 import React from 'react'
 
 import { Button } from 'src/components/buttons'
-import { H1, P } from 'src/components/typography'
+import { H1, P, Info2 } from 'src/components/typography'
+import { ReactComponent as TxOutIcon } from 'src/styling/icons/direction/cash-out.svg'
+import { neon, spacer } from 'src/styling/variables'
 
 const styles = {
   logo: {
     maxHeight: 80,
     maxWidth: 200
   },
   title: {
-    margin: [[24, 0, 32, 0]]
+    margin: 0,
+    marginBottom: 42,
+    textAlign: 'center'
   },
   text: {
     margin: 0
   },
   button: {
-    marginTop: 'auto',
-    marginBottom: 58
+    margin: [[0, 'auto']]
   },
   modalContent: {
     display: 'flex',
     flexDirection: 'column',
-    alignItems: 'center',
-    padding: [[0, 42]],
-    flex: 1
+    justifyContent: 'space-between',
+    flex: 1,
+    paddingBottom: 141,
+    '& > div': {
+      paddingBottom: 72,
+      '& > h1': {
+        color: neon,
+        marginBottom: 12,
+        marginTop: 30,
+        textAlign: 'center',
+        '& > svg': {
```

Comment:
There's no need for the child selectors, just create a new class and put the className on the element you want.